internal/api/user.go

```diff
@@ -144,3 +144,38 @@ func (h *handler) DeleteUser(c *gin.Context) {
 
 	c.JSON(http.StatusOK, response.SuccessResponse(nil))
 }
+
+func (h *handler) ChangePassword(c *gin.Context) {
```

Comment:
```suggestion
func (h *handler) PutMe(c *gin.Context) {
```

Would suggest to name the handler like the restful api route. Maybe we will extend this route in future (changing E-Mail, 2FA, ...)